task/net/src/bsp/gimlet_bc.rs

```diff
@@ -2,6 +2,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#[cfg(any(not(feature = "ksz8463"), not(feature = "mgmt")))]
```

Comment:
I don't feel strongly about it, but this could be De Morgan'd into
```suggestion
#[cfg(not(all(feature = "ksz8463", feature = "mgmt"))]
```
which is slightly shorter

Reply:
Yeah, I agree that DeMorganing this seems right.